modules/cluster/addons/aws-ebs-csi-driver.yaml

```diff
@@ -924,7 +924,7 @@ spec:
         env:
         - name: CSI_ENDPOINT
           value: unix:/csi/csi.sock
-        image: k8s.gcr.io/provider-aws/aws-ebs-csi-driver:v0.9.0
+        image: k8s.gcr.io/provider-aws/aws-ebs-csi-driver:latest
```

Comment:
The use of the `latest` tag caught my eye here. The `latest` image tag is built from the `master` branch of https://github.com/kubernetes-sigs/aws-ebs-csi-driver/

I note we're referencing the `alpha` overlay [here](https://github.com/cookpad/terraform-aws-eks/blob/master/modules/cluster/addons/kustomize/overlays/aws-ebs-csi-driver/kustomization.yaml#L4)

This includes `base` which does specify `latest`.

If we update our kustomization to use the `stable` overlay, we'll return to using the `v0.9.0` tag, as defined [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/v0.9.0/deploy/kubernetes/overlays/stable/kustomization.yaml#L7)

Reply:
>The use of the latest tag caught my eye here. 

Yes, I have the same opinion and I don't want to use the latest tag in this module.

Using `stable` overlay is the best choice if we don't support the `alpha` feature 

>Starting from Kubernetes 1.17, CSI migration is supported as beta feature (alpha since 1.14). If you have persistence volumes that are created with in-tree kubernetes.io/aws-ebs plugin, you could migrate to use EBS CSI driver. To turn on the migration, set CSIMigration and CSIMigrationAWS feature gates to true for kube-controller-manager and kubelet.

Reply:
@dan-slinky-ckpd 

[Accroding to the official document](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html), EKS 1.17 does not support the CSI migration feature and the later version didn't say they support the feature.

>EKS has not enabled the CSIMigrationAWS feature flag. This will be enabled in a future release, along with detailed migration instructions. For more info on CSI migration, see the Kubernetes blog.

So, +1 to the stable + `v0.9.0` tag. WDYT?

Reply:
👍 `stable`

Reply:
when I added this to the module we were on alpha because beta didn't support volume resizing.

It would be good to check the status on that!

Reply:
Ok... volume resizing is beta since 1.16 so LGTM!